addons/pos_self_order/static/src/app/pages/product_list_page/product_list_page.js

```diff
@@ -127,4 +128,11 @@ export class ProductListPage extends Component {
             },
         });
     }
+
+    getProductInfo(product_id) {
+        this.dialog.add(ProductInfoPopup, {
+            product: this.selfOrder.productByIds[product_id],
+            title: _t("%s", this.selfOrder.productByIds[product_id].name),
+        });
+    }
```

Comment:
So yes, this function do not have any requirement to be in ProductList, so like already said, put it in product_card

Reply:
also this is not translating anything

Reply:
I think, Product name should already be translated by backend